src/test/kotlin/calculator/StringAddCalculatorTest.kt

```diff
@@ -0,0 +1,63 @@
+package calculator
+
+import calculator.domain.Expression
+import calculator.domain.StringAddCalculator
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatExceptionOfType
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.EmptySource
+import org.junit.jupiter.params.provider.ValueSource
+
+internal class StringAddCalculatorTest {
+    private lateinit var calculator: StringAddCalculator
+
+    @BeforeEach
+    fun setUp() {
+        calculator = StringAddCalculator()
+    }
```

Comment:
이건 취향일 수 있는데 저는 아래와 같이 선언하는 편입니다.

```suggestion
    private val calculator: StringAddCalculator = StringAddCalculator()
```

lateinit과 가변(var)을 쓰지 않아도 되며 테스트간 격리도 똑같이 일어나고 BeforeEach와 같은 어노테이션도 쓸 필요가 없어서 저는 위와 같은 코드가 더 편하더라구요.